tests/codegen/float/algebraic.rs

```diff
@@ -0,0 +1,146 @@
+// Verify that algebraic intrinsics generate the correct LLVM calls
+
+#![crate_type = "lib"]
+#![feature(f16)]
+#![feature(f128)]
+#![feature(float_algebraic)]
+
+// CHECK-LABEL: @f16_algebraic_add
+#[no_mangle]
+pub fn f16_algebraic_add(a: f16, b: f16) -> f16 {
+    // CHECK: fadd reassoc nsz arcp contract half {{(%a, %b)|(%b, %a)}}
```

Comment:
Rust and LLVM are allowed to have intermediate ops or for whatever reason `fadd` something other than `%a` and `%b` directly. To avoid future noise, could you change all the tests to just match any vars rather than exact? The important part is just that we are adding the modifiers.

```suggestion
    // CHECK: fadd reassoc nsz arcp contract half %{{.*}}, %{{.*}}
```